client/src/components/ToolsTab.tsx

```diff
@@ -297,6 +304,102 @@ const ToolsTab = ({
                     );
                   },
                 )}
+                <div className="pb-4">
+                  <div className="flex items-center justify-between mb-2">
+                    <h4 className="text-sm font-semibold">Meta:</h4>
```

Comment:
This should say "Tool-specific Metadata:" to disambiguate between these pairs and those entered on the `Metadata` tab